rllib/algorithms/algorithm_config.py

```diff
@@ -145,6 +130,12 @@ class AlgorithmConfig(_Config):
         ...     ).fit()
     """
 
+    @staticmethod
```

Comment:
Moved this here as it might be useful to have in a central place (instead of somewhere hidden in the c'tor).

Reply:
nice